tests/Platforms/SQLServerPlatformTest.php

```diff
@@ -134,6 +134,24 @@ public function testGeneratesTypeDeclarationsForStrings(): void
         );
     }
 
+    public function testGeneratesTypeDeclarationsForAsciiStrings(): void
```

Comment:
Please also add an assertion where `'fixed' => false` is used to prevent regressions when code changes in the future.

Reply:
Should I also add the assertion to `testGeneratesTypeDeclarationsForStrings`?

Reply:
Yes, please.